core-bundle/contao/languages/en/tl_templates.xlf

```diff
@@ -56,6 +56,9 @@
       <trans-unit id="tl_templates.compare">
         <source>Compare "%s" with another template</source>
       </trans-unit>
+      <trans-unit id="tl_templates.twig_studio_hint">
+        <source>If you want to create or editor Twig templates, use the %s instead.</source>
```

Comment:
```suggestion
        <source>If you want to create or edit Twig templates, use the %s instead.</source>
```

Reply:
Why a `%s`? If `Template Studio` is a name, it will never be spelled any other way and if not, it should be spelled `Template studio`.